2.6/test/run

```diff
@@ -14,6 +14,7 @@ IMAGE_NAME=${IMAGE_NAME-centos/mongodb-26-centos7-candidate}
 CIDFILE_DIR=$(mktemp --suffix=mongodb_test_cidfiles -d)
 
 function cleanup() {
+    set +e
```

Comment:
this was something i put in over the weekend in an attempt to debug what was going on, before i got the point of running the push job against my own fork.  agree it is probably not needed, should not be in 2.4 either.